resources/content/config-content/haproxy/apply.sh

```diff
@@ -9,9 +9,15 @@ awk '/BEGIN.*PRIVATE KEY/{i++}{print > "/etc/haproxy/certs/server"i".pem"}' /etc
 rm /etc/haproxy/certs/certs.pem
 if [ ! -s /etc/haproxy/certs/default.pem ]; then rm /etc/haproxy/certs/default.pem;fi
 
-if ! haproxy -v | grep -q 'version 1.5'; then
-    echo deb http://archive.ubuntu.com/ubuntu trusty-backports main universe > /etc/apt/sources.list.d/backports.list
-    apt-get update && apt-get install -y haproxy -t trusty-backports
+
+
+if ! haproxy -v | grep -q 'version 1.6'; then
+    # save config file as update to haproxy 1.6 overrides it
+    cp /etc/haproxy/haproxy.cfg /etc/haproxy/haproxy.cfg_tmp
+    apt-get update -y && apt-get install -y software-properties-common \
+    && add-apt-repository -y ppa:vbernat/haproxy-1.6 \
+    && apt-get update -y && apt-get install -y haproxy
```

Comment:
Looks like apt-get install -y haproxy doesn't respect "answer all" flag (it prompts to reload the upgraded config), therefore it gets stuck:

```
root@68f3c4b9dee6:/# tail -f /var/log/apt/term.log
Configuration file '/etc/haproxy/haproxy.cfg'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** haproxy.cfg (Y/I/N/O/D/Z) [default=N] ? 
```

will see what we can do